src/main/java/com/snowflake/kafka/connector/config/ConnectorConfigDefinition.java

```diff
@@ -461,6 +461,13 @@ public static ConfigDef getConfig() {
             ENABLE_TASK_FAIL_ON_AUTHORIZATION_ERRORS_DEFAULT,
             ConfigDef.Importance.LOW,
             "If set to true the Connector will fail its tasks when authorization error from"
-                + " Snowflake occurred");
+                + " Snowflake occurred")
+        .define(
+            ICEBERG_ENABLED,
+            ConfigDef.Type.BOOLEAN,
+            ICEBERG_ENABLED_DEFAULT_VALUE,
+            ConfigDef.Importance.MEDIUM,
```

Comment:
i would say it;s somewhere between medium and high. It requires completely different table setup done by the customer

Reply:
Sure, I can set it to high. I was looking for some explanation what does it mean but unfortunately it's completely arbitrary.